### DISCUSSION TITLE
What is a preset?

### DISCUSSION THREAD

**Discussion by manuganji** [tw-in-js/twind#304] [Q&A] [answered]
Hi, sorry I read through the docs but I can barely understand it. What does a preset do? I don't see it in the docs for [configuration](https://twind.dev/handbook/configuration.html) but I see presets used in the [Svelte Kit example](https://github.com/tw-in-js/twind/blob/next/examples/with-sveltekit/src/twind.config.ts) so I'm curious.

**Comment by IgnusG** [comment, marked as answer]
Twind, in v1 at least, works more like unocss in that it doesn’t come with any predefined rules (it’s basically an empty CSS compiler and you can write your own rules if you’d like).

if you’d like to use some predefined ones such as the Tailwind 3.0 ones you can include the Tailwind preset and those rules will become available to you. 

Twind v0 comes with the tailwind 2.0 rules preloaded I think so you don’t need to specify anything there. I think it doesn’t support presets at all which is why it’s missing in the documentation (the docs are not up to date yet as v1 is still in alpha).

Until the release you can consult examples, https://github.com/tw-in-js/twind/blob/next/website/pages/docs/reference.md and https://github.com/tw-in-js/twind/blob/next/website/pages/docs/migration.md#notable-changes for any new behaviour.

**Reply by manuganji** [reply]
Thank you for this answer, @IgnusG. It makes sense. :)
